doc/design.md

```diff
@@ -127,13 +127,13 @@ The solution is to have the bridge wait until a publisher becomes available befo
 
 Due to possible delays induced by the bridge, it is possible that the original deadline of a publisher may not be met when forwarding to a different domain.
 Furthermore, if deadline is important for the user application, then it is probably better for them to set manually on the bridge.
-Therefore, by default the bridge will assume a maximum value for deadline, but allow users to opt-in to matching the publisher value or setting the value manually.
+Therefore, by default the bridge will assume the RMW defined default value for deadline (typically implying "no deadline"), but allow users to opt-in to matching the publisher value or setting the value manually.
```

Comment:
"Therefore, by default the bridge" could be: "Therefore, by default, the bridge"
I suggest "RMW-defined" versus "RMW defined" 

Is this sentence saying that if the user doesn't specify a value for the deadline then instead of using the publisher's value it will use the RMW-defined default value instead? This is how I understand it after reading it a couple of times, but I'm not 100% sure I understand it correctly. Perhaps it could be something like:

"Therefore, by default, the bridge will use the RMW-defined default value for deadlines. Users may direct to use the original value used by the publisher or another value instead by setting..."

Reply:
Yes, that's correct: If no value is provided by the user, then the RMW-defined value is used. I've broken this down into multiple sentences as suggestion (thanks!). Is it more clear?

Reply:
Yes, it's clear now.